src/PrestaShopBundle/Form/Admin/Sell/Product/Stock/StockType.php

```diff
@@ -80,28 +80,17 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('packed_products', EntitySearchInputType::class, [
+            ->add('packed_products', ProductSearchType::class, [
                 'label' => $this->trans('Pack of products', 'Admin.Catalog.Feature'),
                 'label_tag_name' => 'h2',
+                'limit' => 0,
```

Comment:
Maybe it's worth forcing the option to search combinations, even if it's the default at least it makes it clear (it could almost be a required option to avoid any misusage)